dayton/spiders/reap_spider.py

```diff
@@ -56,11 +58,19 @@ def getTaxEligibility(self, response):
     def getPaymentPlan(self, response):
         sel = Selector(response)
         item = response.meta['item']
-        item['paymentplan'] = False
-        for paymentplan in sel.xpath('//*[@id="content_middle_wide"]/div/form/div/tr/td/font/font/b/text()').extract():
-            item['paymentplan'] = 'Unapplied Payments:' in paymentplan or item['paymentplan']
+        item['payment_plan'] = False
+        
+        for paymentXPath in sel.xpath('//*[@id="content_middle_wide"]/div[3]/form/table/tr'):
+            try:
+                data = paymentXPath.xpath('./td/text()').extract()
+                # if not an empty list
+                if data:
+                    date = datetime.strptime(data[0].strip(), '%m-%d-%Y')
+                    payment =  Decimal(re.sub(r'[^\d.]', '', data[1]))
+                    if date.year > datetime.today().year - 3 and payment > 0:
+                        item['payment_plan'] = True
```

Comment:
I like this approach better than my own. :+1: Let me run it to make sure I get similar results, and then I'll merge it.